.gitignore

```diff
@@ -8,6 +8,9 @@
 # Test binary, built with `go test -c`
 *.test
 
+# elastic-package binary built with go build
```

Comment:
If you use "go build", then most likely you can produce the binary in the repo folder.